config/config/sc-config.yaml

```diff
@@ -190,6 +190,16 @@ prometheus:
     tolerations: []
     affinity: {}
     nodeSelector: {}
+  grafana:
```

Comment:
While you are at it. Please add resources, tolerations, and affinity here. Preferably with defaults for the resource requests.

Reply:
Thanks for adding resources, though I'm still missing tolerations and affinity. Though it's not a blocker if you have a reason not to add it.

Reply:
Right, I missed that you wrote that. Yea, I'll add it